dydx-testnet-4/genesis.json

```diff
@@ -0,0 +1,6642 @@
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx1arrlapn93ysd3hle9z4apsm55l0qxzflvxqrmy",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx1awsudh3flrl6gat67errx0jd68czzceln79mak",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx17mvn3czvy0gpcrxt0k6y5lppfwmwaucvy7n3e3",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx1lzmwnljjmyfr0v25eeu7e6mh5scqxuth8s4peq",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx1lrcfjnz7pmjfmzm624egpdt49w49wawejfpuw0",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx1ltyc6y4skclzafvpznpt2qjwmfwgsndp458rmp",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "10000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx1ldal0sqjf80lcepacdmgtgycunaxn9axwyf34l",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "dydx1lukgmdjlv8elel3syg6n3kg2dxe4x67n3gucc3",
+          "coins": [
+            {
+              "denom": "adv4tnt",
+              "amount": "1000000000000000000000000"
+            }
+          ]
+        }
+      ],
+      "supply": [],
+      "denom_metadata": [
+        {
+          "description": "The native token of the network",
+          "denom_units": [
+            {
+              "denom": "adv4tnt",
+              "exponent": 0,
+              "aliases": []
+            },
+            {
+              "denom": "dv4tnt",
+              "exponent": 18,
+              "aliases": []
+            }
+          ],
+          "base": "adv4tnt",
+          "display": "dv4tnt",
+          "name": "dYdX V4 Testnet Token",
+          "symbol": "dv4tnt",
+          "uri": "",
+          "uri_hash": ""
+        }
+      ],
+      "send_enabled": []
+    },
+    "blocktime": {
+      "params": {
+        "durations": [
+          "300s",
+          "1800s"
+        ]
+      }
+    },
+    "bridge": {
+      "acknowledged_event_info": {
+        "eth_block_height": 4322136,
+        "next_id": 5
```

Comment:
is this correct? IIUC, @tqin7 wanted to test recognizing some events at genesis and then recognizing them after. 

for example, if there are 20 events on the contract, this value would be set so that 10 events are recognized at genesis and 10 would be recognized as the network catch up on the eth blocks

Reply:
@BrendanChou and @tqin7 to confirm

Reply:
also, if we are trying to recognize some events at genesis, should we have created bank balances?